modules/net-lb-int/variables.tf

```diff
@@ -186,6 +188,12 @@ variable "name" {
   type        = string
 }
 
+variable "override_name" {
+  description = "Optional override name to specify exact name without changes."
+  type        = string
+  default     = null
+}
+
```

Comment:
If I read correctly, this is no longer used, is it?

Reply:
Correct, moved into each individual forwarding rule now.